Templates/CSharp/Base/IEntityRequestBuilder.Base.template.tt

```diff
@@ -119,30 +119,6 @@ public string GetNonCollectionRequestBuilderProperty(OdcmProperty odcmProperty)
     return this.GetRequestBuilderProperty(propertyName, propertyRequestBuilder);
 }
 
-
-// -------------------------------------------------------------
-// Methods to create an OData method property definition within an entity class
-// -------------------------------------------------------------
-public string GetMethodRequestBuilderMethod(
```

Comment:
Moved to RequestBuilder.Base as this code is shared with CollectionRequestBuilder.